python/docs/source/usage.rst

```diff
@@ -12,99 +14,310 @@ Resolve partitions for current version of the DeltaTable
     >>> dt.version()
     3
     >>> dt.files()
-    ['part-00000-cb6b150b-30b8-4662-ad28-ff32ddab96d2-c000.snappy.parquet', 'part-00000-7c2deba3-1994-4fb8-bc07-d46c948aa415-c000.snappy.parquet', 'part-00001-c373a5bd-85f0-4758-815e-7eb62007a15c-c000.snappy.parquet']
+    ['part-00000-cb6b150b-30b8-4662-ad28-ff32ddab96d2-c000.snappy.parquet', 
+     'part-00000-7c2deba3-1994-4fb8-bc07-d46c948aa415-c000.snappy.parquet', 
+     'part-00001-c373a5bd-85f0-4758-815e-7eb62007a15c-c000.snappy.parquet']
+
 
+Loading a Delta Table
+---------------------
 
-Apply filtering on partitions for current version of the partitioned DeltaTable
+To load the current version, use the constructor:
+
+.. code-block:: python
+
+    >>> dt = DeltaTable("../rust/tests/data/delta-0.2.0")
+
+Alternatively, if you have a data catalog you can load it by reference to a 
+database and table name. Currently only AWS Glue is supported.
+
+.. TODO: auth to data catalog?
```

Comment:
FYI #522 explains the requirements of the Data Catalog integration.

Reply:
Thanks!